### ISSUE TITLE
[VSCode-Proper] A11y_ViennaVsCodeExtension_VisualStudioCodeToolsForAI_Screenreader: Narrator focus is not moving hence narrator is not announcing the page contents like Buttons and links.

### ISSUE THREAD

**Issue by greazer** [Microsoft/vscode#63258]
“Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

Environment Details:
Application Name: ViennaVsCodeExtension
Application Version: 1.28.1
Operating System: Windows10
Url:  https://marketplace.visualstudio.com/items?itemName=ms-toolsai.vscode-ai#overview

Repro Steps:
1) Launch VisualStudio Code.
2) Navigate to Extensions TAB-->Navigate to VisualStudioCodeToolsForAI Extension and hit Enter key.
3) Start Narrator.
4) Verify how Narrator reads contents of the page.

Actual
Narrator focus is not moving hence narrator is not announcing the page contents like Buttons and links.
As keyboard focus is moving properly and all links and buttons are actionable using Keyboard but due to narrator focus issue it is not reading details.

Note:
Same issue is found in following pages:
1. ToolsForAIStartPage: 
    Launch VisualStudio Code-->Go to Command palette and type 'OpenToolsForAIStartPage' and         hit enter.

2.AIOpenSamplePage:
Launch VisualStudio Code-->Go to Command Palette and search AI: Open sample page and hit enter.

Expected
Narrator should announce all the links and buttons details.

User Impact
Screenreader user will not be able to understand page content.

Recommendation
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=1.3.1

https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=1.3.2

MAS Reference
https://microsoft.sharepoint.com/%3Aw%3A/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7D

https://microsoft.sharepoint.com/%3Aw%3A/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B1ee227a7-e3c5-47b8-ab18-48ca8595077b%7D

[704890_A11y_ViennaVsCodeExtension_VisualStudioCodeToolsForAI_Screenreader.zip](https://github.com/Microsoft/vscode/files/2587486/704890_A11y_ViennaVsCodeExtension_VisualStudioCodeToolsForAI_Screenreader.zip)

**Comment by sandy081** [comment]
@isidorn This is also similar... So closing this with your comment from other issue.

Narrator is not supported for vscode.
Accessibility issue from extension should go in extensions repositories. And we already have duplicates of this.
